check.sh

```diff
@@ -0,0 +1,4 @@
+pnpm lint:fix
```

Comment:
I don't think this script needs to be committed. I'd prefer to leave this in the package.json